src/QuestionsForm.js

```diff
@@ -6,35 +6,51 @@ import TextInput from './TextInput'
 import NumberInput from './NumberInput'
 import CheckboxInput from './CheckboxInput'
 import RadioInput from './RadioInput'
+import DateInput from './DateInput'
 import Button from './Button'
 
 export default function QuestionsForm({ onClick }) {
-  const { register, handleSubmit } = useForm()
+  const { register, handleSubmit, watch, errors } = useForm({
+    reValidateMode: 'onSubmit',
+  })
 
-  const onSubmit = (data) => {
+  const onFormSubmit = (data) => {
     console.log(data)
```

Comment:
I think you forgot to remove this console log 😉

Reply:
Haha thanks 😄